OrbitGl/TimeGraph.cpp

```diff
@@ -61,11 +55,9 @@ TimeGraph::~TimeGraph() {
 }
 
 void TimeGraph::SetStringManager(std::shared_ptr<StringManager> str_manager) {
```

Comment:
Since you changed this to a raw pointer in `TrackManager`, wouldn't it be consistent to do the same here?

Reply:
They are different things. I will try to erase StringManager in TimeGraph and TrackManager and make a new one on GpuTrack.